docs/synapse/userguides/ug017_storm_ref_stats.rst

```diff
@@ -1,16 +1,194 @@
-Storm Reference - Statistical Operators
-=======================================
+Storm Reference - Statistical Operator
+======================================
 
-Todo
+The statistical operator is used to generate data about data in Synapse.
```

Comment:
I would change `generate data about data` to something about how it returns calculated statistics about the data.

Reply:
fixed